plugin/trino-mariadb/src/test/java/io/trino/plugin/mariadb/TestMariaDbJdbcConfig.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.mariadb;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+public class TestMariaDbJdbcConfig
+{
+    @Test
+    public void testIsUrlWithoutDatabase()
+    {
+        assertTrue(isUrlWithoutDatabase("jdbc:mariadb://example.net:3306"));
+        assertTrue(isUrlWithoutDatabase("jdbc:mariadb://example.net:3306/"));
```

Comment:
Is there any reason to allow an empty character as valid URL?

Reply:
Thanks for review! Here, I think an empty character is the same as the not specified the database, this most an habit, just like web url, xx.xxx.xx is the same as xx.xxx.xx/ . Also the maria client treat them as the same all return database is null.